### PR TITLE
Add voidroleplay

### PR DESCRIPTION
**Pull request by celaatv**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/voidroleplay/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "nice_name": "VoidRoleplay",
+  "direct_ip": "voidroleplay.de",
+  "server_wildcards": [
+    "%.voidroleplay.de"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://www.voidroleplay.de",
```
Comment (DoJapHD) [inline minecraft_servers/voidroleplay/manifest.json:12]: Not reachable

File: minecraft_servers/voidroleplay/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "voidroleplay",
+  "nice_name": "VoidRoleplay",
+  "direct_ip": "voidroleplay.de",
+  "server_wildcards": [
+    "%.voidroleplay.de"
```
Comment (DoJapHD) [inline minecraft_servers/voidroleplay/manifest.json:6]: Not a real wildcard
https://i.imgur.com/9Kel6GY.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/voidroleplay/manifest.json (hunk 2 above)
Comment (celaatv) [reply minecraft_servers/voidroleplay/manifest.json]: Thank you for noticing 👍, and sorry for my mistakes. The changes are made.

File: minecraft_servers/voidroleplay/manifest.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "voidroleplay",
+  "nice_name": "VoidRoleplay",
+  "direct_ip": "voidroleplay.de",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://voidroleplay.de",
```
Comment (DoJapHD) [inline minecraft_servers/voidroleplay/manifest.json:9]: Website in maintenance mode;
If it still takes some time, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/voidroleplay/manifest.json (hunk 3 above)
Comment (celaatv) [reply minecraft_servers/voidroleplay/manifest.json]: Alright. Removed!

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**